apps/gp2-frontend/src/events/calendar/Calendars.tsx

```diff
@@ -1,10 +1,24 @@
-import { EventsCalendar } from '@asap-hub/react-components';
+import { CalendarList, EventsCalendar } from '@asap-hub/react-components';
 
 import { useCalendars } from './state';
 
 const Calendars: React.FC<Record<string, never>> = () => {
   const { items } = useCalendars();
-  return <EventsCalendar calendars={items} />;
+  return (
+    <EventsCalendar calendars={items}>
+      <CalendarList
+        title="Subscribe to Working Groups on Calendar"
+        description="Below you can find the list of all working groups. Hitting subscribe will allow you to add them to your own personal calendar."
+        calendars={items.filter((calendar) => calendar.projects.length === 0)}
+        hideSupportText
```

Comment:
is the support text correct for us?

Reply:
yes, I think so. 

What have you noticed?

Reply:
just asking, didn't test it